adapter/slack.py

```diff
@@ -28,6 +29,24 @@ def __init__(
     async def handle_events(self, req: Request):  # pragma: no cover
         return await self.handler.handle(req)
 
+    def send_generated_response(
+        self, channel: str, ts: str, engine: ChatEngine, question: str
+    ):
+        try:
+            self.logger.info("generating response")
+
+            chatbot_response = engine.generate_response(query=question)
+
+            self.logger.info("sending generated response")
+
+            self.app.client.chat_update(channel=channel, ts=ts, text=chatbot_response)
+
+        except ChatResponseGenerationError as e:  # pragma: no cover

```

Comment:
Will the error message appear on the Slack app when this error occurs?

Reply:
Yes. But this one is handled properly. The user will see a user-friendly version of it which basically says that an error happened our end

Reply:
What does the user see to know there is an error on our end? Is it a message that can only be seen by the user or in some other way?

Reply:
Yes the response will be ephemeral. They will see this message 

```
Something went wrong when trying to generate your response.
```

You can see it on the lines below it